docs/oscar_references.bib

```diff
@@ -1510,3 +1510,57 @@ @Misc{cohomCalg:Implementation
   year          = {2010},
   url           = {https://github.com/BenjaminJurke/cohomCalg}
 }
+
+@book{Ful97,
+        author = {Fulton, William},
+        note = {With applications to representation theory and geometry},
+        pages = {x+260},
+        publisher = {Cambridge University Press, Cambridge},
+        series = {London Mathematical Society Student Texts},
+        title = {Young tableaux},
+        volume = {35},
+        year = {1997}}
+
+@misc{FLINT,
+        author = {W. Hart and F. Johansson and S. Pancratz},
+        note = {Version 2.4.0, \url{https://flintlib.org}},
```

Comment:
What's the correct reference?

Reply:
The [web page ](https://flintlib.org/authors.html) gives two options, either the one that you were using (yet that web page mentions an outdated version), or
```
@inproceedings{Hart2010,
    author = {W. B. Hart},
    title = {{Fast Library for Number Theory: An Introduction},
    booktitle = {Proceedings of the Third International Congress
        on Mathematical Software},
    series = {ICMS'10},
    year = {2010},
    location = {Kobe, Japan},
    pages = {88--91},
    numpages = {4},
    publisher = {Springer-Verlag},
    address = {Berlin, Heidelberg},
    note = {\url{https://flintlib.org}}
}
```
That one is stable, so maybe that's easier.

Reply:
Okay, I did this in commit 8ebf2fd. But the reference manager seems to not show the URL which I do not find optimal. Also, I used the label FLINT instead of Har12 (or Hart2010), I don't know which is best.